src/internal/m365/collection/drive/collection.go

```diff
@@ -592,6 +605,7 @@ func (oc *Collection) streamDriveItem(
 	// Drive content download requests are also rate limited by graph api.
 	// Ensure that this request goes through the drive limiter & not the default
 	// limiter.
+	// TODO: seems like we could do this earlier in the collection processing.
```

Comment:
This TODO will go away after rebase with the rate limiter ctx binding changes :)

Reply:
sounds good, I'll remove it.